homeassistant/components/dormakaba_dkey/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the Dormakaba dKey integration."""
+
+DOMAIN = "dormakaba_dkey"
+
+DEVICE_TIMEOUT = 30
+UPDATE_SECONDS = 120
+
+ASSOCIATION_DATA = "association_data"
```

Comment:
Prefix with `CONF_` in constants for config keys.
```suggestion
CONF_ASSOCIATION_DATA = "association_data"
```